packages/cli/src/databases/entities/workflow-entity.ts

```diff
@@ -35,6 +35,9 @@ export class WorkflowEntity extends WithTimestampsAndStringId implements IWorkfl
 	@Column()
 	active: boolean;
 
+	@Column({ default: false })
+	isArchived: boolean;
```

Comment:
Can we add a docline? It's not immediately obvious what "archiving" means if you're not familiar with the feature.